go/proxyd/backend.go

```diff
@@ -247,7 +251,37 @@ func (b *Backend) Forward(ctx context.Context, reqs []*RPCReq, isBatch bool) ([]
 			),
 		)
 
-		res, err := b.doForward(ctx, reqs, isBatch)
+		var (
+			res        []*RPCRes
+			err        error
+			defaultRes []*RPCRes
+			defaultErr error
+			debugRes   []*RPCRes
+			debugErr   error
+
+			resultMethods = []string{"eth_gasPrice", "eth_estimateGas", "eth_call", "eth_getBalance", "eth_chainId"}
+		)
+		defaultRes, defaultErr = b.doForward(ctx, reqs, isBatch, false)
+		if b.debugRpcURL != "" {
+			// We return the value of the debug endpoint if it is available
+			debugRes, debugErr = b.doForward(ctx, reqs, isBatch, true)
```

Comment:
```suggestion
		defaultRes, defaultErr = b.doForward(ctx, reqs, isBatch, false)
		if b.debugRpcURL != "" {
			// We return the value of the debug endpoint if it is available
			debugRes, debugErr = b.doForward(ctx, reqs, isBatch, true)
```

the parameters don't match here (false, true)